dhis2-android-app/src/main/java/org/dhis2/mobile/ui/activities/DataEntryActivity.java

```diff
@@ -204,12 +204,18 @@ public void onLoadFinished(Loader<Form> loader, Form form) {
     public void onLoaderReset(Loader<Form> loader) {
     }
 
-    private void setupToolbar() {
+    private void setupToolbar(String title) {
```

Comment:
if when Toolbar is initialized we don´t know the title, I think better if we create a method ChangeTitle that is invoked when the title is known because now FindById for R.id.toolbar is realized twice and is not the optimal and SetupToolBar method that not change the title.